### ISSUE TITLE
About the value of normVal

### ISSUE THREAD

**Issue by gaowu9595** [sacmehta/ESPNet#51]
Hi,
As said in the code that 'normalization value, as defined in ERFNet paper'. So I read the paper (Efficient Residual Factorized ConvNet for Real-time Semantic) , but I didn't find something about it. Then I went to see the source code of ERFNet, as shown below. The value of weight is fixed...Can you tell me more about the origin of this formula?Thank you

![image](https://user-images.githubusercontent.com/15248829/51516788-7ffe4580-1e53-11e9-8ae9-6d1efd046588.png)

**Comment by sacmehta** [comment]
Check here:

https://github.com/Eromera/erfnet/blob/master/train/model/loss_general.lua

ERFNets original code was written in lua and then translated to PyTorch. While translating, i assume they just extracted class imbalance weights from cached files in lua and used directly (I assume they did it to save time).
